src/components/GrayBG.js

```diff
@@ -1,13 +1,22 @@
-import styled from 'styled-components'
+import styled, { css } from 'styled-components'
 import remcalc from 'remcalc'
 import is from 'styled-is'
 import breakpoint from 'styled-components-breakpoint'
 
 // topOffset prop should be a negative number for overlapping
 // the gray background
 
-// offsetBottom is a boolean that defines that the offset
-// is at the bottom of the section rather than the top
+const afterStyles = css`
+  content: ' ';
+  width: 100%;
+  bottom: 0;
+  background-color: white;
+  display: block;
+  position: absolute;
+  z-index: 2;
+`
+
+const PADDING = 50
```

Comment:
This was done so that we get the top offset working. If not padding is added things will be glued to the top.

This is way the cleanest way me and zooey could make it work because of the top offset.

That 50 is a placeholder number

Also it's only used here so I don't think it makes sense to be in the theme because it will be removed when all the top offsets are removed

Reply:
> If not padding is added things will be glued to the top.

When we use the `GreyBackground` component but don't use the `topOffset` prop (e.g. on the homepage), then the `margin` values default to `50`

### With PADDING=50
![image](https://user-images.githubusercontent.com/15656538/50694888-f8cb6800-1032-11e9-9a1c-139b24fc62b4.png)
![image](https://user-images.githubusercontent.com/15656538/50694757-aab66480-1032-11e9-9f21-c296cc5018cd.png)

### Without 50
![image](https://user-images.githubusercontent.com/15656538/50694856-e3eed480-1032-11e9-86aa-ff0b98cc37f2.png)
![image](https://user-images.githubusercontent.com/15656538/50694822-d0436e00-1032-11e9-9f2d-926a62a7d238.png)

So I'm not quite sure what you mean by 'glued to the top'?

I would have thought that, in the situation where you _don't_ offset the grey background, then the spacing between the grey section & the white section should be defined by padding within each respective component?

Reply:
Hmm....it's not really very easy to tell what I mean from those pictures. Especially since the grey is so faint. Happy to get on a call if that would be easier?

Basically, I'm wondering whether we could have the default `GreyBackground` component (without extra props added), being nothing other than this line:
https://github.com/yldio/yld.io/blob/db965ea319d7e8ff4976d84fec3e92dd1838543b/src/components/GreyBG.js#L22

My thinking is that, we only need the following extras if/when we add the `topOffset` prop:
- `position: absolute`
- `position: relative` on children
- z-index values
- margin/padding alterations

Right? Otherwise, `GreyBG` can behave exactly like the [BlueBG component](https://github.com/yldio/yld.io/blob/db965ea319d7e8ff4976d84fec3e92dd1838543b/src/components/BlueBG.js)?